templates/docs/examples/patterns/suru/dark.html

```diff
@@ -0,0 +1,11 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Suru / Dark{% endblock %}
+
+{% block standalone_css %}patterns_suru{% endblock %}
+
+{% set is_paper = True %}
+{% block content %}
+<div class="p-strip--dark" style="background: #262626;">
```

Comment:
It's just for the purpose of the demo (you don't need the strip itself for suru obviously) and inline background is needed for standalone examples (that don't contain strip styles), but we can work out a better solution to avoid inline styles I guess.